test/app/lib/crypto/address.spec.ts

```diff
@@ -0,0 +1,12 @@
+import {getMasterKey} from "./helpers"
+import * as PublicKey from "../../../../app/lib/crypto/key/publicKey"
+import {encode} from "../../../../app/lib/crypto/address/p2pkh"
+
+describe("p2pkh", () => {
+  const masterKey = getMasterKey("000102030405060708090a0b0c0d0e0f")
+
+  it("should encode address", () => {
```

Comment:
`it("should derive a Bech32-encoded P2PKH address from an extended public key", () => { `